pkg/api/mlflow/service/artifact/storage/local_test.go

```diff
@@ -52,13 +49,9 @@ func TestGetArtifact_Error(t *testing.T) {
 	storage, err := NewLocal(nil)
 	assert.Nil(t, err)
 
-	file, err := storage.Get(filepath.Join(runArtifactRoot), "some-other-item")
+	file, err := storage.Get(filepath.Join(runArtifactRoot), "non-existent-file")
```

Comment:
could you please remove the useless call to `filepath.Join`, as indicated in my previous comment?

Reply:
check